### PR TITLE
feat: alert component

### PR DESCRIPTION
**Pull request by harshitadaswani**
alert component added

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.success-outline {
+    background-color: #f3f3f3;
+    color: var(--success-alert);
+    border: 0.1rem solid var(--success-alert);
+}
+
+.warning-outline {
+    background-color: #f3f3f3;
```
Comment (MaruthiPothuganti) [inline components/alert.css]: Background color seems to be common. You can use a variable for that

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended: using CSS variables for repeated colors.

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    border-radius: 0.4rem;
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap;
+    gap: 1rem;
+    margin: var(--s-size) 0;
+}
+
+.icon {
+    color: #f3f3f3;
```
Comment (krishnakant01) [inline components/alert.css]: would recommend storing the repeating values in a variable and using it.

**Comment by krishnakant01** [comment]
Apart from that, it looks good to merge.

**Pull request merged**